composables/playlist.ts

```diff
@@ -50,3 +50,32 @@ export function addTrackToPlaylist(playlistId: number, trackId: number) {
     link: [`</track/${trackId}>,`],
   });
 }
+
+export function addAlbumToPlaylist(playlistId: number, albumId: number) {
```

Comment:
If we care about following the Nuxt conventions, these three functions are not composables,  but utils, and should be in `utils/playlist.ts`.

https://vuejs.org/guide/reusability/composables.html#what-is-a-composable

Reply:
we actually don't need those functions anymore